timepiece/crm/views.py

```diff
@@ -3084,3 +3085,36 @@ def project_download_attachment(request, project_id, attachment_id):
         return HttpResponseRedirect(project_attachment.get_download_url())
     except:
         return HttpResponse('Project attachment could not be found.')
+
+def get_minding(request):
+    user = request.user
+    projects = Project.objects.filter(
+        point_person=user, status=4)[:3]
+    projects_json = []
+    for project in projects:
+        projects_json.append({
```

Comment:
You don't have to change this, but I would consider putting this as a `to_json` method on the `class Project`.  Just in case we needed this serialization elsewhere.  But you don't have to move it for me to approve.